rundeckapp/grails-app/conf/spring/resources.groovy

```diff
@@ -812,14 +813,16 @@ beans={
 
     //provider implementations
     tokenDataProvider(GormTokenDataProvider)
+    userDataProvider(GormUserDataProvider)
 
 
     //manager setup
     rundeckDataManager(BaseDataManager)
```

Comment:
All usages and references to the DataManger need to be removed from this file.

Reply:
Done